### ISSUE TITLE
Title bar issue

### ISSUE THREAD

**Issue by ounos** [wilfm/GnomeExtensionMaximusTwo#31]
When I switch among different instances of sublime/firefox/whatever, the title is not refreshed, it keeps displaying the old title.

OS: Ubuntu 14.04

**Comment by wilfm** [comment]
Yes it happens with libreoffice too in newer versions... I'll investigate and see if I can find a fix
